packages/flutter/lib/src/material/divider.dart

```diff
@@ -36,37 +36,53 @@ import 'theme.dart';
 /// ![](https://flutter.github.io/assets-for-api-docs/assets/material/divider.png)
 ///
 /// ```dart
-/// Widget build(BuildContext context) {
-///   return Center(
-///     child: Column(
-///       children: <Widget>[
-///         Expanded(
-///           child: Container(
-///             color: Colors.amber,
-///             child: const Center(
-///               child: Text('Above'),
-///             ),
-///           ),
-///         ),
-///         const Divider(
-///           color: Colors.black,
-///           height: 20,
-///           thickness: 5,
-///           indent: 20,
-///           endIndent: 0,
-///         ),
-///         Expanded(
-///           child: Container(
-///             color: Colors.blue,
-///             child: const Center(
-///               child: Text('Below'),
-///             ),
-///           ),
-///         ),
-///       ],
-///     ),
-///   );
-/// }
+///   Widget build(BuildContext context) {
+///    return Center(
+///      child: Column(
+///        children: <Widget>[
+///          Expanded(
+///            child: Container(
+///              color: Colors.amber,
+///              child: const Center(
+///                child: Text('Above'),
+///              ),
+///            ),
+///          ),
+///          const Divider(
+///            color: Colors.black,
+///            height: 20,
+///            thickness: 5,
+///            indent: 20,
+///            endIndent: 20,
+///          ),
+///          // Subheader example from Material spec.
+///          // https://material.io/components/dividers#types
+///          Container(
+///            padding: EdgeInsets.only(left: 20),
+///            child: Align(
+///              alignment: AlignmentDirectional.centerStart,
+///              child: Text(
+///                'Subheader',
+///                style: Theme.of(context).textTheme.bodyText2.copyWith(
+///                  fontSize: 12.0,
+///                  color: Theme.of(context).textTheme.caption.color,
+///                ),
+///                textAlign: TextAlign.start,
+///              ),
+///            ),
+///          ),
+///          Expanded(
+///            child: Container(
```

Comment:
Similar comments here: lets show developers how to configure this in a way that works with dark mode, by using the color scheme.

Reply:
Done